<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1488539410225" stop="1488539427380">
	<name>Validate Footer</name>
	<title>Validate Footer</title>
	<test-cases>
		<test-case start="1488539410226" stop="1488539426556" status="passed">
			<name>Loading Home Page </name>
			<title>Loading Home Page </title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="9cbfd2cd-af38-407a-b2ff-104b9b834b82-attachment.png" type="image/png" size="322367"/>
			</attachments>
		</test-case>
		<test-case start="1488539426556" stop="1488539426786" status="passed">
			<name>Validate Correct page Loaded</name>
			<title>Validate Correct page Loaded</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="8e5fecb1-8c0f-414b-a466-460756274a51-attachment.png" type="image/png" size="314307"/>
			</attachments>
		</test-case>
		<test-case start="1488539426787" stop="1488539427153" status="passed">
			<name>Getting number of footer links</name>
			<title>Getting number of footer links</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="e775f651-b6bf-4aed-bf9d-384f5387eed1-attachment.png" type="image/png" size="314307"/>
			</attachments>
		</test-case>
		<test-case start="1488539427153" stop="1488539427379" status="failed">
			<name>encountered a declaration exception</name>
			<title>encountered a declaration exception</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="bd0a2f16-fedc-4c5e-92fa-61b5a0684f78-attachment.png" type="image/png" size="314306"/>
			</attachments>
			<failure>
				<message>Error: &apos;expect&apos; was used when there was no current spec, this could be because an asynchronous test timed out</message>
				<stack-trace>Error: &apos;expect&apos; was used when there was no current spec, this could be because an asynchronous test timed out
    at Env.expect (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:938:15)
    at expect (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3415:18)
    at global.expect (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:193:10)
    at Suite.&lt;anonymous&gt; (E:\protractorcms\cyborg\features\FT\FT-001\FT-001.js:38:17)
    at addSpecsToSuite (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:833:25)
    at Env.describe (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:802:7)
    at describe (C:\Users\Ravikiran\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3375:18)
    at Object.&lt;anonymous&gt; (E:\protractorcms\cyborg\features\FT\FT-001\FT-001.js:10:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>